frontend/public/components/dashboards-page/overview-dashboard/top-consumers-card.tsx

```diff
@@ -78,9 +107,29 @@ const TopConsumersCard_: React.FC<DashboardItemProps> = ({
   const topConsumersResult = prometheusResults.getIn([currentQuery, 'result']);
 
   const stats = getInstantVectorStats(topConsumersResult, topConsumersType.metric, metricTypeSort.humanize);
-
   const data = topConsumersType.mutator ? topConsumersType.mutator(stats) : stats;
 
+  const top5Data = [];
+  const consumersLoaded = _.get(resources, ['consumers', 'loaded']);
+  const consumersLoadError = _.get(resources, ['consumers','loadError']);
+  const consumersData = _.get(resources, ['consumers', 'data']) as K8sResourceKind[];
+  if (consumersLoaded && !consumersLoadError) {
+    data.some(d => {
```

Comment:
`some` is used to check whether at least one element matches the specified test, but we're not referencing the returned (boolean) value here.

Isn't it better to use `forEach` here?

Reply:
I cannot return early from `forEach` loop and I only need to find first 5 resources. `some` allows it by returning `true`.  However I see that its a bit weird usage so I replaced the loop with `for..of`